layouts/error.vue

```diff
@@ -1,11 +1,77 @@
 <template>
   <v-app>
-    <h1 v-if="error.statusCode === 404">
-      {{ pageNotFound }}
-    </h1>
-    <h1 v-else>
-      {{ otherError }}
-    </h1>
+    <v-row v-if="error.statusCode === 404">
```

Comment:
You can add a parent element `<v-container>` which will make the consistent the margin spacing to the left/right side of the viewport.